src/main/java/hu/oe/nik/szfmv/automatedcar/AutomatedCar.java

```diff
@@ -40,4 +41,14 @@ private void calculatePositionAndOrientation() {
         this.setPosY(this.getPosY() + (int)movingVector.getY());
         setRotation(pt.getAutoSzoge());
     }
+
+    @Override
+    public AffineTransform getTransform(int offsetX, int offsetY) {
```

Comment:
Offset nélkül legyen, az offsetet majd kint megoldják

Reply:
Ha átírom így:
```Java
AffineTransform t = object.getTransform(0, 0);
t.translate(t.getTranslateX() + offsets[0], t.getTranslateY() + offsets[1]);
```
Akkor az alábbi képet kapom:
![kép](https://user-images.githubusercontent.com/10576653/66722418-749de080-ee0d-11e9-94a0-76f4e6c32163.png)

Szóval nem értek egyet ezzel. Számít a sorrendje a transzformációnak és esetünkben nem lehet utólag eltolni még egyszer.

Reply:
a megjelenítésből most össze-vissza van az eltolás, mert eddig azt hittük, ti natúrban adjátok
az hogy hol legyen eltolva nagyban függ attól, hogy kit fog kérdezni a sávtartás meg az ütközés detektor: ha titeket (már pedig gondolom így lesz), akk szerintem nálatok is kellene eltolás